{"version":"3.8.0","messages":["\nAn error occurred while loading ./spec/hello_spec.rb.\n\u001b[31mFailure/Error: \u001b[0m\u001b[32myield\u001b[0m(name)\u001b[0m\n\u001b[31m\u001b[0m\n\u001b[31mLocalJumpError:\u001b[0m\n\u001b[31m  no block given (yield)\u001b[0m\n\u001b[36m# ./lib/hello.rb:4:in `block in hello_t'\u001b[0m\n\u001b[36m# ./lib/hello.rb:2:in `each'\u001b[0m\n\u001b[36m# ./lib/hello.rb:2:in `hello_t'\u001b[0m\n\u001b[36m# ./lib/hello.rb:8:in `<top (required)>'\u001b[0m\n\u001b[36m# ./spec/spec_helper.rb:1:in `require_relative'\u001b[0m\n\u001b[36m# ./spec/spec_helper.rb:1:in `<top (required)>'\u001b[0m\n\u001b[36m# ./spec/hello_spec.rb:1:in `<top (required)>'\u001b[0m\n","No examples found."],"examples":[],"summary":{"duration":0.000261427,"example_count":0,"failure_count":0,"pending_count":0,"errors_outside_of_examples_count":2},"summary_line":"0 examples, 0 failures, 2 errors occurred outside of examples"}